mlir/test/Target/LLVMIR/openmp-llvm.mlir

```diff
@@ -2820,27 +2823,18 @@ llvm.func @task(%arg0 : !llvm.ptr) {
   llvm.return
 }
 // CHECK-LABEL: @task..omp_par
```

Comment:
Here, I think we should also check for the block structure of `@task(...)` itself to test private op inlining.